### DISCUSSION TITLE
Current time in Update flow operation

### DISCUSSION THREAD

**Discussion by filiphanes** [directus/directus#17175] [Archive]
Hi,
I would like to update datetime field to current time in flow, but when I put following liquid template to payload it evaluates to undefined and then failes to update in db. In Liquid Playground it works nice.
```json
{
    "confirmed_at": "{{'now' | date: '%Y-%m-%d %H:%M:%S'}}"
}
```
How could I update current timestamp?

**Comment by paescuj** [comment]
There's no support for Liquid template. Please look here for supported variables and expressions: https://docs.directus.io/configuration/flows.html#data-chain-variables

(Out of curiosity: What made you feel that Liquid is supported?)

The simplest method to update the date in a field when the item is updated is to use the "On Update" setting:
<img width="600" alt="Screenshot 2023-01-17 at 13 21 13" src="https://user-images.githubusercontent.com/5363448/212898022-e520275e-df8c-4f95-8a9e-d85d2a0f2a05.png">

If that is not an option for you (you might want to update the field only in specific cases), then I recommend to obtain the current time via the "Run Script" operation: https://docs.directus.io/configuration/flows/operations.html#run-script

**Reply by filiphanes** [reply]
> (Out of curiosity: What made you feel that Liquid is supported?)
when I used this, it was filled with ip

Email templates are in Liquid, so I thought that all substitutions are made by this library.

And this is working.
```json
{
    "confirm_ip": "{{$accountability.ip}}",
}
```
anyway, thanks, I will use Run script operation
